Rotator/RotatorCollection.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Zenstruck\BackupBundle\Rotator;
+
+use ArrayIterator;
+use Countable;
+use IteratorAggregate;
+
+final class RotatorCollection implements Countable, IteratorAggregate, Rotator
```

Comment:
Ok, this is rotator collection, so pre/post rotations can be decided with multiple rotators.